docs/examples/UserGuide/interpolation.jl

```diff
@@ -0,0 +1,32 @@
+# # Using Interpolation On The Fly
+
+using Tyler, GLMakie
+using Interpolations: interpolate, Gridded, Linear
+
+f(lon,lat)=cosd(16*lon)+sind(16*lat)
+
+f_in_0_1_range(lon,lat)=0.5+0.25*f(lon,lat)
+
+nodes=(-180.0:180.0, -90.0:90.0)
+array=[f(lon,lat) for lon in nodes[1], lat in nodes[2]]
+array=(array.-minimum(array))./(maximum(array)-minimum(array))
+itp = interpolate(nodes, array, Gridded(Linear()))
+cols=Makie.to_colormap(:viridis)
+col(i)=RGBAf(Makie.interpolated_getindex(cols,i))
```

Comment:
Couldn't these be part of `Interpolator`? It seems like some unnecessary work to set it up.

They could be keywords if people did need to customise?